engine/frontend/src/utils/download.js

```diff
@@ -0,0 +1,34 @@
+const DEFAULT_ELEMENT_NAME = "a"
+const DEFAULT_FILE_TYPE = "text/plain"
+
+export const saveTextAsFile = (text, fileName, options = {}) => {
```

Comment:
@Peeeekay can't you just say: `elementName="box"`  rather than having this generic options hashmap with the associated code below? A default parameter is straight forward to do.

Reply:
the method will look like: saveTextAsFile(text, fileName, fileType="", elementName="") .. so no good way of distinguish between fileType and/or elementName. ( or I couldn;t find any).. this seems to be more cleaner.